### ISSUE TITLE
Controller not working in Moonlight, but working in TV's UI

### ISSUE THREAD

**Issue by gschintgen** [OneLiberty/moonlight-chrome-tizen#42]
One of my controllers is not working in Moonlight, neither in the app's UI nor during gameplay. It's a SNES-type USB controller: "Buffalo Classic USB Gamepad". On a Linux PC it's detect by `lsusb` as follows:
```
Bus 002 Device 005: ID 0583:2060 Padix Co., Ltd (Rockfire) 2-axis 8-button gamepad
```

The gamepad is working fine in Samsung's TV UI though (QN90B series). It just stops working as soon as I have launched the Moonlight app using that very same controller.
The controller is supported by Moonlight-Qt (both on a Pi4 and on an ordinary x64 mini pc running Linux).

Hopefully it's just a missing mapping or so.

Is it possible to retrieve logs from the TV? (I think I saw somewhere that it's possible, but now I can no longer find any info on that.)
edit: found it. https://github.com/OneLiberty/moonlight-chrome-tizen/discussions/9

**Comment by gschintgen** [comment]
The app is based on Chromium, right? Including gamepad support?

It's noteworthy that there's even a direct reference to my gamepad in Chromium's source code:
https://github.com/chromium/chromium/blob/b38bf28a85a29d0b3879ec328e8959b5b1f3c5be/device/gamepad/gamepad_id_list.h#L104
and in Chrome (Ubuntu Linux) it's indeed working fine.

**Comment by OneLiberty** [comment]
No we're reliant on samsung gamepad support https://developer.samsung.com/smarttv/develop/guides/user-interaction/gamepad.html and mostly https://www.w3.org/TR/gamepad/
Most XInput gamepads should work, but i have no list to provide. 
Without having acces to the specific gamepad it's hard to diagnose and fix with my limited knowledge. 

Is the gamepad detected in the logs ? 

Maybe you can test it with https://wpt.live/gamepad/

**Comment by Wubbiez** [comment]
Any update on this? My Switch Pro controller and a 8bitdo lite 2 are unable to work once Moonlight launches. I launch the moonlight app with both of the controllers fine. Once in the moonlight app they are no longer responsive.

**Comment by gschintgen** [comment]
> Any update on this? My Switch Pro controller and a 8bitdo lite 2 are unable to work once Moonlight launches. I launch the moonlight app with both of the controllers fine. Once in the moonlight app they are no longer responsive.

No, I did not yet have time to extract the logs. I don't think there can be progress without them.

**Comment by hotpotatodude** [comment]
sorry to bother but any updates on this? my joycons also dont work :(

**Comment by Tripton** [comment]
Any updates on this? Or any recommendations which controller does work?

**Comment by OneLiberty** [comment]
No update on this, i can only suggest you getting a Xbox One/ series controller or a dualshock4/dualsense 
Those works in the app
